stepxx/README.md

```diff
@@ -11,7 +11,6 @@ The following are the modifications made to the code after the last official ste
 ## Task ??: Make the frame rate configurable
 ## Task ??: Split main.go into multiple files
 ## Task ??: Refactor Player and Ghosts as Sprites
-## Task ??: Add support for multiple player lives
```

Comment:
@danicat, BTW, noticed your latest screenshot with Live emojis as PowerUps. Maybe you wanted to add to the tutorial functionality to increase lives on player contact with these objects? And as this wasn't yet implemented in step09, this line shouldn't be removed from stepxx?

Reply:
Nope, those were not designed to be extra lives. It is just a placeholder for the power up pills. The proper "emoji" would be a big square, but I didn't look that great for demos because the lack of contrast.

Step XX is where I play with stuff. It is not part of the tutorial and should not be considered "canon". The tutorial may or may not reach that state at some point in the future, but for now I use it mostly for demoing the potential of the tutorial.